packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -51,18 +52,37 @@ export const withReactQuery = (Wrapped, options = {}) => {
             const queryClient = (res.locals.__queryClient =
                 res.locals.__queryClient || new QueryClient(queryClientConfig))
 
+            res.__performanceTimer.mark(PERFORMANCE_MARKS.reactQueryPrerender, 'start')
             // Use `ssrPrepass` to collect all uses of `useQuery`.
             await ssrPrepass(appJSX)
-
+            res.__performanceTimer.mark(PERFORMANCE_MARKS.reactQueryPrerender, 'end')
```

Comment:
Out of the scope of the PR, but should we handle potential errors thrown by ssrPrepass? 

```
res.__performanceTimer.mark(PERFORMANCE_MARKS.reactQueryPrerender, 'start')

try {
        // Use `ssrPrepass` to collect all uses of `useQuery`
        await ssrPrepass(appJSX)
} catch (error) {
        logger.error(`Error during ssrPrepass: ${error}`, {namespace: 'withReactQuery.doInitAppState'})
} finally {
        res.__performanceTimer.mark(PERFORMANCE_MARKS.reactQueryPrerender, 'end')
}
```

Reply:
I'm going to leave this for future as the app is going to crash really bad regardless if we catch it or not...